modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/item/ContentDashboardItem.java

```diff
@@ -77,7 +77,7 @@ public ContentDashboardItemAction getDefaultContentDashboardItemAction(
 
 	public static class Version {
 
```

Comment:
We don't need to sort by number or string. Just show the approved version first and the draft (if any) second.